### DISCUSSION TITLE
404 error on repo.securityonion.net for bind-libs etc

### DISCUSSION THREAD

**Discussion by Masaya-A** [Security-Onion-Solutions/securityonion#4059] [Unsupported Versions] [answered]
Version: 2.3.50
Install source: network from CentOS 7 minimum
Install type: Standalone

```
[kinoko@yama ~]$ sudo yum install bind-utils
Loaded plugins: versionlock
Excluding 4 updates due to versionlock (use "yum versionlock status" to show them)
Resolving Dependencies
--> Running transaction check
---> Package bind-utils.x86_64 32:9.11.4-26.P2.el7_9.5 will be installed
--> Processing Dependency: bind-libs-lite(x86-64) = 32:9.11.4-26.P2.el7_9.5 for package: 32:bind-utils-9.11.4-26.P2.el7_9.5.x86_64
--> Processing Dependency: bind-libs(x86-64) = 32:9.11.4-26.P2.el7_9.5 for package: 32:bind-utils-9.11.4-26.P2.el7_9.5.x86_64
--> Processing Dependency: liblwres.so.160()(64bit) for package: 32:bind-utils-9.11.4-26.P2.el7_9.5.x86_64
--> Processing Dependency: libisccfg.so.160()(64bit) for package: 32:bind-utils-9.11.4-26.P2.el7_9.5.x86_64
--> Processing Dependency: libisc.so.169()(64bit) for package: 32:bind-utils-9.11.4-26.P2.el7_9.5.x86_64
--> Processing Dependency: libirs.so.160()(64bit) for package: 32:bind-utils-9.11.4-26.P2.el7_9.5.x86_64
--> Processing Dependency: libdns.so.1102()(64bit) for package: 32:bind-utils-9.11.4-26.P2.el7_9.5.x86_64
--> Processing Dependency: libbind9.so.160()(64bit) for package: 32:bind-utils-9.11.4-26.P2.el7_9.5.x86_64
--> Processing Dependency: libGeoIP.so.1()(64bit) for package: 32:bind-utils-9.11.4-26.P2.el7_9.5.x86_64
--> Running transaction check
---> Package GeoIP.x86_64 0:1.5.0-14.el7 will be installed
--> Processing Dependency: geoipupdate for package: GeoIP-1.5.0-14.el7.x86_64
---> Package bind-libs.x86_64 32:9.11.4-26.P2.el7_9.5 will be installed
--> Processing Dependency: bind-license = 32:9.11.4-26.P2.el7_9.5 for package: 32:bind-libs-9.11.4-26.P2.el7_9.5.x86_64
---> Package bind-libs-lite.x86_64 32:9.11.4-26.P2.el7_9.5 will be installed
--> Running transaction check
---> Package bind-license.noarch 32:9.11.4-26.P2.el7_9.5 will be installed
---> Package geoipupdate.x86_64 0:2.5.0-1.el7 will be installed
--> Finished Dependency Resolution
--> Finding unneeded leftover dependencies
Found and removing 0 unneeded dependencies

Dependencies Resolved

============================================================================================================================================================================================================================
 Package                                               Arch                                          Version                                                           Repository                                      Size
============================================================================================================================================================================================================================
Installing:
 bind-utils                                            x86_64                                        32:9.11.4-26.P2.el7_9.5                                           updates                                        260 k
Installing for dependencies:
 GeoIP                                                 x86_64                                        1.5.0-14.el7                                                      base                                           1.5 M
 bind-libs                                             x86_64                                        32:9.11.4-26.P2.el7_9.5                                           updates                                        157 k
 bind-libs-lite                                        x86_64                                        32:9.11.4-26.P2.el7_9.5                                           updates                                        1.1 M
 bind-license                                          noarch                                        32:9.11.4-26.P2.el7_9.5                                           updates                                         91 k
 geoipupdate                                           x86_64                                        2.5.0-1.el7                                                       base                                            35 k

Transaction Summary
============================================================================================================================================================================================================================
Install  1 Package (+5 Dependent packages)

Total size: 3.1 M
Total download size: 1.6 M
Installed size: 7.5 M
Is this ok [y/d/N]: y
Downloading packages:
Delta RPMs disabled because /usr/bin/applydeltarpm not installed.
bind-libs-9.11.4-26.P2.el7_9.5 FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/bind-libs-9.11.4-26.P2.el7_9.5.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not Found                                 ]  0.0 B/s |    0 B  --:--:-- ETA
Trying other mirror.
To address this issue please refer to the below wiki article

https://wiki.centos.org/yum-errors

If above article doesn't help to resolve this issue please use https://bugs.centos.org/.

bind-libs-lite-9.11.4-26.P2.el FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/bind-libs-lite-9.11.4-26.P2.el7_9.5.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not Found                            ]  0.0 B/s |    0 B  --:--:-- ETA
Trying other mirror.
bind-license-9.11.4-26.P2.el7_ FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/bind-license-9.11.4-26.P2.el7_9.5.noarch.rpm: [Errno 14] HTTPS Error 404 - Not Found                              ]  0.0 B/s |    0 B  --:--:-- ETA
Trying other mirror.
bind-utils-9.11.4-26.P2.el7_9. FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/bind-utils-9.11.4-26.P2.el7_9.5.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not Found                                ]  0.0 B/s |    0 B  --:--:-- ETA
Trying other mirror.


Error downloading packages:
  32:bind-libs-9.11.4-26.P2.el7_9.5.x86_64: [Errno 256] No more mirrors to try.
  32:bind-utils-9.11.4-26.P2.el7_9.5.x86_64: [Errno 256] No more mirrors to try.
  32:bind-libs-lite-9.11.4-26.P2.el7_9.5.x86_64: [Errno 256] No more mirrors to try.
  32:bind-license-9.11.4-26.P2.el7_9.5.noarch: [Errno 256] No more mirrors to try.

[kinoko@yama ~]$
```

This causes some packages which should be installed at same time are not installed.
For my environment during fresh installing, httpd-tools, net-tools, ntpdate, tcpdump and vim-enhanced are not installed. Then I got some errors (ex. Wazuh's ini.sls uses netstat) or some functions are not working (ex. BPF).

**Comment by TOoSmOotH** [comment, marked as answer]
Can you give it a try now? I was able to re-produce but now it's not failing anymore.

**Reply by Masaya-A** [reply]
It seems fine now.
Thank you!

**Reply by stondino00** [reply]
Still an issue.  Happening for me.



=============================================================================================================================================================
 Package                                       Arch                        Version                                       Repository                     Size
=============================================================================================================================================================
Installing:
 kernel                                        x86_64                      3.10.0-1160.25.1.el7                          updates                        50 M
Updating:
 bind-export-libs                              x86_64                      32:9.11.4-26.P2.el7_9.5                       updates                       1.1 M
 bind-libs                                     x86_64                      32:9.11.4-26.P2.el7_9.5                       updates                       157 k
 bind-libs-lite                                x86_64                      32:9.11.4-26.P2.el7_9.5                       updates                       1.1 M
 bind-license                                  noarch                      32:9.11.4-26.P2.el7_9.5                       updates                        91 k
 bind-utils                                    x86_64                      32:9.11.4-26.P2.el7_9.5                       updates                       260 k
 device-mapper                                 x86_64                      7:1.02.170-6.el7_9.5                          updates                       297 k
 device-mapper-event                           x86_64                      7:1.02.170-6.el7_9.5                          updates                       192 k
 device-mapper-event-libs                      x86_64                      7:1.02.170-6.el7_9.5                          updates                       192 k
 device-mapper-libs                            x86_64                      7:1.02.170-6.el7_9.5                          updates                       325 k
 firewalld                                     noarch                      0.6.3-13.el7_9                                updates                       449 k
 firewalld-filesystem                          noarch                      0.6.3-13.el7_9                                updates                        51 k
 glibc                                         x86_64                      2.17-324.el7_9                                updates                       3.6 M
 glibc-common                                  x86_64                      2.17-324.el7_9                                updates                        12 M
 kernel-tools                                  x86_64                      3.10.0-1160.25.1.el7                          updates                       8.1 M
 kernel-tools-libs                             x86_64                      3.10.0-1160.25.1.el7                          updates                       8.0 M
 kexec-tools                                   x86_64                      2.0.15-51.el7_9.2                             updates                       351 k
 lvm2                                          x86_64                      7:2.02.187-6.el7_9.5                          updates                       1.3 M
 lvm2-libs                                     x86_64                      7:2.02.187-6.el7_9.5                          updates                       1.1 M
 mdadm                                         x86_64                      4.1-7.el7_9                                   updates                       440 k
 nss                                           x86_64                      3.53.1-7.el7_9                                updates                       869 k
 nss-sysinit                                   x86_64                      3.53.1-7.el7_9                                updates                        66 k
 nss-tools                                     x86_64                      3.53.1-7.el7_9                                updates                       535 k
 openldap                                      x86_64                      2.4.44-23.el7_9                               updates                       356 k
 python-firewall                               noarch                      0.6.3-13.el7_9                                updates                       355 k
 python-perf                                   x86_64                      3.10.0-1160.25.1.el7                          updates                       8.1 M
Removing:
 kernel                                        x86_64                      3.10.0-1160.21.1.el7                          @updates                       64 M

Transaction Summary
=============================================================================================================================================================
Install   1 Package
Upgrade  25 Packages
Remove    1 Package

Total download size: 99 M
Is this ok [y/d/N]: y
Downloading packages:
No Presto metadata available for updates
bind-export-libs-9.11.4-26.P2. FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/bind-export-libs-9.11.4-26.P2.el7_9.5.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not Found
Trying other mirror.
To address this issue please refer to the below wiki article

https://wiki.centos.org/yum-errors

If above article doesn't help to resolve this issue please use https://bugs.centos.org/.

bind-libs-9.11.4-26.P2.el7_9.5 FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/bind-libs-9.11.4-26.P2.el7_9.5.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not FoundETA
Trying other mirror.
bind-libs-lite-9.11.4-26.P2.el FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/bind-libs-lite-9.11.4-26.P2.el7_9.5.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not Found
Trying other mirror.
bind-license-9.11.4-26.P2.el7_ FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/bind-license-9.11.4-26.P2.el7_9.5.noarch.rpm: [Errno 14] HTTPS Error 404 - Not Found
Trying other mirror.
bind-utils-9.11.4-26.P2.el7_9. FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/bind-utils-9.11.4-26.P2.el7_9.5.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not FoundTA
Trying other mirror.
device-mapper-1.02.170-6.el7_9 FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/device-mapper-1.02.170-6.el7_9.5.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not FoundA
Trying other mirror.
device-mapper-event-1.02.170-6 FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/device-mapper-event-1.02.170-6.el7_9.5.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not Found
Trying other mirror.
device-mapper-event-libs-1.02. FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/device-mapper-event-libs-1.02.170-6.el7_9.5.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not Found
Trying other mirror.
device-mapper-libs-1.02.170-6. FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/device-mapper-libs-1.02.170-6.el7_9.5.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not Found
Trying other mirror.
(1/26): firewalld-0.6.3-13.el7_9.noarch.rpm                                                                                           | 449 kB  00:00:00
(2/26): firewalld-filesystem-0.6.3-13.el7_9.noarch.rpm                                                                                |  51 kB  00:00:00
glibc-2.17-324.el7_9.x86_64.rp FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/glibc-2.17-324.el7_9.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not Found --:--:-- ETA
Trying other mirror.
glibc-common-2.17-324.el7_9.x8 FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/glibc-common-2.17-324.el7_9.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not Found-- ETA
Trying other mirror.
kernel-3.10.0-1160.25.1.el7.x8 FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/kernel-3.10.0-1160.25.1.el7.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not Found-- ETA
Trying other mirror.
kernel-tools-3.10.0-1160.25.1. FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/kernel-tools-3.10.0-1160.25.1.el7.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not Found
Trying other mirror.
kernel-tools-libs-3.10.0-1160. FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/kernel-tools-libs-3.10.0-1160.25.1.el7.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not Found
Trying other mirror.
kexec-tools-2.0.15-51.el7_9.2. FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/kexec-tools-2.0.15-51.el7_9.2.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not Found ETA
Trying other mirror.
lvm2-2.02.187-6.el7_9.5.x86_64 FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/lvm2-2.02.187-6.el7_9.5.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not Found:--:-- ETA
Trying other mirror.
lvm2-libs-2.02.187-6.el7_9.5.x FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/lvm2-libs-2.02.187-6.el7_9.5.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not Found- ETA
Trying other mirror.
(3/26): mdadm-4.1-7.el7_9.x86_64.rpm                                                                                                  | 440 kB  00:00:00
nss-3.53.1-7.el7_9.x86_64.rpm  FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/nss-3.53.1-7.el7_9.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not FoundB  --:--:-- ETA
Trying other mirror.
nss-sysinit-3.53.1-7.el7_9.x86 FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/nss-sysinit-3.53.1-7.el7_9.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not Found:-- ETA
Trying other mirror.
nss-tools-3.53.1-7.el7_9.x86_6 FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/nss-tools-3.53.1-7.el7_9.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not Found--:-- ETA
Trying other mirror.
openldap-2.4.44-23.el7_9.x86_6 FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/openldap-2.4.44-23.el7_9.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not Found--:-- ETA
Trying other mirror.
(4/26): python-firewall-0.6.3-13.el7_9.noarch.rpm                                                                                     | 355 kB  00:00:00
python-perf-3.10.0-1160.25.1.e FAILED
https://repo.securityonion.net/file/securityonion-repo/updates/Packages/python-perf-3.10.0-1160.25.1.el7.x86_64.rpm: [Errno 14] HTTPS Error 404 - Not FoundA
Trying other mirror.


Error downloading packages:
  32:bind-libs-9.11.4-26.P2.el7_9.5.x86_64: [Errno 256] No more mirrors to try.
  nss-tools-3.53.1-7.el7_9.x86_64: [Errno 256] No more mirrors to try.
  32:bind-libs-lite-9.11.4-26.P2.el7_9.5.x86_64: [Errno 256] No more mirrors to try.
  7:device-mapper-event-1.02.170-6.el7_9.5.x86_64: [Errno 256] No more mirrors to try.
  7:lvm2-2.02.187-6.el7_9.5.x86_64: [Errno 256] No more mirrors to try.
  7:lvm2-libs-2.02.187-6.el7_9.5.x86_64: [Errno 256] No more mirrors to try.
  kexec-tools-2.0.15-51.el7_9.2.x86_64: [Errno 256] No more mirrors to try.
  32:bind-export-libs-9.11.4-26.P2.el7_9.5.x86_64: [Errno 256] No more mirrors to try.
  glibc-common-2.17-324.el7_9.x86_64: [Errno 256] No more mirrors to try.
  7:device-mapper-libs-1.02.170-6.el7_9.5.x86_64: [Errno 256] No more mirrors to try.
  glibc-2.17-324.el7_9.x86_64: [Errno 256] No more mirrors to try.
  kernel-3.10.0-1160.25.1.el7.x86_64: [Errno 256] No more mirrors to try.
  7:device-mapper-event-libs-1.02.170-6.el7_9.5.x86_64: [Errno 256] No more mirrors to try.
  nss-sysinit-3.53.1-7.el7_9.x86_64: [Errno 256] No more mirrors to try.
  32:bind-license-9.11.4-26.P2.el7_9.5.noarch: [Errno 256] No more mirrors to try.
  kernel-tools-3.10.0-1160.25.1.el7.x86_64: [Errno 256] No more mirrors to try.
  32:bind-utils-9.11.4-26.P2.el7_9.5.x86_64: [Errno 256] No more mirrors to try.
  nss-3.53.1-7.el7_9.x86_64: [Errno 256] No more mirrors to try.
  7:device-mapper-1.02.170-6.el7_9.5.x86_64: [Errno 256] No more mirrors to try.
  kernel-tools-libs-3.10.0-1160.25.1.el7.x86_64: [Errno 256] No more mirrors to try.
  python-perf-3.10.0-1160.25.1.el7.x86_64: [Errno 256] No more mirrors to try.
  openldap-2.4.44-23.el7_9.x86_64: [Errno 256] No more mirrors to try.

**Reply by stondino00** [reply]
Ok seems to be working again now.
